app/controllers/api/v0/admin/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V0
+    module Admin
+      class UsersController < ApiController
+        before_action :authenticate_with_api_key_or_current_user!
+        before_action :authorize_super_admin
+        skip_before_action :verify_authenticity_token, only: %i[create]
+
+        def create
+          User.invite!(user_params)
```

Comment:
We can add an initing user here, e.g. `User.invite!(current_user, user_params)`. Not sure if we want that in the general case though so I didn't add it yet.